ansible/roles/csm.sbps.dns_srv_records/files/sbps_dns_srv_records.sh

```diff
@@ -34,6 +34,7 @@ hsn_a_records=""
 nmn_a_records=""
 
 system_name="$(cat /etc/environment | grep SYSTEM_NAME | awk -F= '{print $2;}')"
+site_domain="$(cat /etc/environment | grep SITE_DOMAIN | awk -F= '{print $2;}')"
```

Comment:
This could be simplified to just:
```
eval $(grep -e SITE_DOMAIN -e SYSTEM_NAME /etc/environment)
```
and then use `$SYSTEM_NAME` and `$SITE_DOMAIN` instead of `$system_name` and `$site_domain`

At a minimum, get rid of the `cat` piped to `grep`, just give grep the file name:
```
site_domain="$(grep SITE_DOMAIN /etc/environment ...
```